src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -67,13 +68,13 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         try {
             $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, true);
         } catch (\Exception $e) {
-            $this->logException($exception, sprintf('Exception thrown when handling an exception (%s: %s)', get_class($e), $e->getMessage()), false);
+            $this->logException($e, sprintf('Exception thrown when handling an exception (%s: %s at %s line %s)', get_class($e), $e->getMessage(), $e->getFile(), $e->getLine()), false);
```

Comment:
What about something as simple as this? The context should log the Exception and the formatters will properly format the logs for you. In case of the HTML formatter, this will be the basic exception information; Exception Class, Message and Trace.
```php
// message is up for improvement
$this->logException($e, 'An Exception has been thrown when handling another Exception.', false);
```